src/expr/src/table_function/mod.rs

```diff
@@ -62,6 +64,7 @@ pub fn build_from_prost(
         Generate => new_generate_series(prost, chunk_size),
         Unnest => new_unnest(prost, chunk_size),
         RegexpMatches => new_regexp_matches(prost, chunk_size),
+        Range => new_range(prost, chunk_size),
```

Comment:
What about change `new_generate_series` to `new_generate_series<const STOP_INCLUSIVE: bool>`,
and then
```suggestion
        Generate => new_generate_series::<false>(prost, chunk_size),
        Unnest => new_unnest(prost, chunk_size),
        RegexpMatches => new_regexp_matches(prost, chunk_size),
        Range => new_generate_series::<true>(prost, chunk_size),
```

Then `range.rs` becomes unnecessary.

Reply:
ok